tasks/main.yml

```diff
@@ -13,8 +13,8 @@
 #   notify:
 #     - enable ufw
 
-- include: ssl.yml
+- import_tasks: ssl.yml
```

Comment:
Hey. Thanks for the PR.

Not sure if you are still following them but in this case this wouldn't work. In a loop we should use `include_tasks`